lib/wallet/src/Cardano/Wallet/DB/Store/Transactions/TransactionInfo.hs

```diff
@@ -27,7 +27,7 @@ import Cardano.Wallet.DB.Store.Transactions.Model
 import Cardano.Wallet.Primitive.Slotting
     ( TimeInterpreter, interpretQuery, slotToUTCTime )
 import Cardano.Wallet.Primitive.Types.Tx
-    ( TxCBOR, TxMeta (..) )
```

Comment:
Another `TxMeta` is imported a few lines above, maybe this was why it wasn't caught by CI 🤷‍♂️

Reply:
interesting ghc 9.2 has better support for redundant constraint. I hope it will help us avoiding the -Wno-redundant-constraints on the new `Excluding` / `Including`  constraints.